.circleci/config.yml

```diff
@@ -1000,6 +1000,11 @@ workflows:
           requires:
             - build-distros-linux
       # Run acceptance tests using the docker image built for the control plane
+      - cleanup-gcp-resources
+      - acceptance-gke-1-20:
+          requires:
+            - cleanup-gcp-resources
+            - dev-upload-docker
```

Comment:
Remove this before merging. This is only to ensure that the acceptance tests run against a public cloud.